app/dashboard/_components/cards/InclusionScore/InclusionScore.tsx

```diff
@@ -28,8 +29,8 @@ const InclusionScore = ({
   benchmark = 50,
 }: InclusionScoreProps) => {
   return (
-    <Card classname='col-span-full lg:col-span-4 h-min flex items-center flex-col lg:flex-row lg:items-start'>
-      <div className='flex max-w-[220px] flex-col gap-8 lg:flex-col '>
+    <Card classname='relative col-span-full lg:col-span-4 h-min flex items-center flex-col lg:flex-row lg:items-start'>
+      <div className='flex min-w-[220px] flex-col gap-8 lg:flex-col '>
```

Comment:
should this really have a this big min with, is it responsive this way?

Reply:
This is why i need it in testing, I said this when working with it on tuesday, that i have to test this, when i add the chart to the card, it will push the text to 2 lines, and the card will not be full with reaching next card and cards filling out the screen. 
So when i changed the max with to min, it stretched out, we have to test it and if there is a better way we can then fix, maybe there is a better way

Reply:
why cant you test the responsiveness in your own branch? the develop branch will make no changes to your code. if it's because you're missing the first part of this card why not merge it in from main?

Reply:
I have and it worked... I ment with all cards... sorry for bad explanation

Reply:
Jag visade ju igår och då sa du att vi ändå måste testa alla kort sen, när jag satt brevid dig, du sa det ser rätt ut, det var med min-width